101-aks/azuredeploy.json

```diff
@@ -99,7 +99,7 @@
             "apiVersion": "2018-03-31",

```

Comment:
There is a newer version 2019-02-01.  See https://docs.microsoft.com/en-us/azure/templates/microsoft.containerservice/2019-02-01/managedclusters

Reply:
There's still a couple of refinements coming there, it just went out last week. Would rather stick with 2018-03-31 at least for next couple of weeks.

Reply:
Make sense.